conveyor/slot/signal.go

```diff
@@ -15,3 +15,9 @@
  */
 
 package slot
+
+const (
```

Comment:
What about added type to this constants, use stringer and iota?

Reply:
not now